### DISCUSSION TITLE
where is the automatic letterbox config located?

### DISCUSSION THREAD

**Discussion by tonho911** [smplayer-dev/smplayer#879] [Q&A] [answered]
where is the automatic letterbox config located?
I mean by using letterbox
apps will display subtitles in the mailbox space if possible.
Subtitles not only cover the video but also increase visibility.

apps will set the height of the mailbox automatically
so that the subtitles are arranged in rows.

what a shame to waste mailbox space and I also found this feature in mpv 

noted. video by movist pro

https://github.com/smplayer-dev/smplayer/assets/148846306/d9f5c30e-40fe-41f0-a5f9-e659a5cdf673

**Comment by smplayer-dev** [comment, marked as answer]
You can use the option Video -> Filters -> Add black borders.
Or you can enable the option "Add black borders in fullscreen" in Preferences -> General -> Video.

**Reply by tonho911** [reply]
you can enable the option "Add black borders in fullscreen" in Preferences -> General -> Video, this is the answer, thank you bro
